tests/Export/Adapters/OptionAdapterTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\Shopware6Common\Tests\Export\Adapters;
+
+use FINDOLOGIC\Shopware6Common\Export\Adapters\OptionsAdapter;
+use FINDOLOGIC\Shopware6Common\Export\Config\PluginConfig;
+use FINDOLOGIC\Shopware6Common\Tests\Traits\AdapterHelper;
+use FINDOLOGIC\Shopware6Common\Tests\Traits\ProductHelper;
+use PHPUnit\Framework\TestCase;
+use Vin\ShopwareSdk\Data\Uuid\Uuid;
+
+class OptionAdapterTest extends TestCase
+{
+    use ProductHelper;
+    use AdapterHelper;
+
+    public OptionsAdapter $optionsAdapter;
+    public PluginConfig $pluginConfig;
+    public function setUp(): void
+    {
+        $this->pluginConfig = $this->getPluginConfig();
+        $this->optionsAdapter = $this->getOptionsAdapter($this->pluginConfig);
+    }
+
+    public function testGetOptionAttributes(): void
```

Comment:
There are no tests for the `VariantConfigurationAdapter`.

It's important to test the different `variantListingConfigs`.